terraform/backend.tf

```
@@ -1,8 +1,18 @@
 terraform {
   backend "azurerm" {
+    # These values can be provided via:
+    # 1. Backend configuration file (-backend-config)
+    # 2. Environment variables (ARM_ACCESS_KEY for storage account key)
+    # 3. Service Principal authentication
+
     resource_group_name  = "terraform-state-mgmt"
     storage_account_name = "aistatemgmt"
     container_name       = "team4-backend"
```

```suggestion
    container_name       = "terraform-tfstate-ai"
```
Please use existing one

---

This^